spec/beaker-pe/pe-client-tools/executable_helper_spec.rb

```diff
@@ -1,5 +1,6 @@
 require 'spec_helper'
 require 'beaker'
+require 'scooter'
```

Comment:
@lucywyman @tvpartytonight beaker-pe is a public repository, but scooter is private and internal to puppetlabs.  By including it in the specs, the specs can't be run without access to the gem. I believe beaker-pe is public because there is some use of beaker to test PE installs outside of puppetlabs.  I'm not certain if that still applies, @kevpl may know.

Probably should replace use of the Scooter::HttpDispatchers::ConsoleDispatcher with a double(:credentials => double(:login => 'foo', :password => 'bar'), :acquire_token_with_credentials => 'baz'), or resolve the public/private discrepancy between beaker-pe/scooter.